machine-config-daemon.spec

```diff
@@ -20,7 +20,9 @@ BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang >= 1
 %autosetup -Sgit -n machine-config-operator-%{commit}
 
 %build
-env VERSION_OVERRIDE=%{version} SOURCE_GIT_COMMIT=%{commit} make daemon
+# By default go build doesn't uses vendored packags with Go modules
+# Should be fixed with Go 1.14 - https://github.com/golang/go/issues/33848
+env VERSION_OVERRIDE=%{version} SOURCE_GIT_COMMIT=%{commit} GOFLAGS='-mod=vendor' WHAT='machine-config-daemon' ./hack/build-go.sh
```

Comment:
Any reason not to set `GOFLAGS` in the `Makefile` instead, so it applies to all builds?

Reply:
Makefile already uses GOFLAGS at https://github.com/openshift/machine-config-operator/blob/master/Makefile#L9 which are used locally to the Makefile and are not exported , adding here really doesn't hep much since it doesn't get passed to hack/build-go.sh  . If I export existing GOFLAGS with additional `-mod=vendor` in  Makefile, that doesn't get well received by build-go.sh script.

Also, I though we might want to pull in latest dependencies defined in go.mod since vendor maybe sometime outdated.